packages/terra-core-docs/src/terra-dev-site/doc/switch/AccessibilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,178 @@
+
+## Why is the accessibility of Terra Switch important? 
+
+The accessibility of a switch is imperative because a switch is a control users use to turn a setting on or off. If switches are not accessible, it can prevent users from understanding what state the switch is in, or worse, prevent the users from interacting with the control. By implementing switches in an accessible way, we are ensuring as many users as possible can use the switch regardless of how they may interact with it (e.g., with a mouse, touch, speech input, keyboard only, or other assistive technology).  
+
+It is critical that a switch is written to be keyboard accessible. Additionally, a switch must be programmatically understood as a switch so that users of assistive technologies can understand what type of a component they are on and how to interact with it. Users expect a switch to turn on or off a setting and not to do anything unexpected. For a switch to be highly usable and easily understood, it is critical that all switches be uniquely labeled and descriptive of what action(on/off) will be taken when activated. For example:  
+
+- **Blind screen reader users** do not necessarily interact with pages in a linear fashion and can use a switch out of context. They need to know what kind of a component they are on and how to interact with It; different components have different expectations. They need the switch's label to be descriptive of its purpose.
+- **People using speech input software** require unique Switch labels to effectively activate the Switch. 
+- **Keyboard only users** navigate the page using only a keyboard. They expect a switch to be keyboard accessible and able to be activated using specific keys. Additionally, visible keyboard focus is crucial for this user to see where they are on the page.
+
+## Accessibility Considerations
+
+### Code Considerations
+
+- Ensure all switches receive keyboard focus and can be only interacted through the keyboard.
+  - Do NOT override keyboard focus state or use a TABINDEX greater than zero. 
+  - Switch should be coded in the users logical reading order of the page to ensure a meaningful experience.  
+- Do NOT override the implicit/explicit role of a switch if another role is required do NOT use the Terra switch.
+
+| Bad Practice | Good Practice |
+|---|---|
+|``` <Switch role="checkbox" isChecked={value} labelText="Show Patients List in Menu" onChange={setValue} /> ```|```<Switch isChecked={value} labelText="Show Patients List in Menu" onChange={setValue} /> ```|
+
+<Notice variant="important" ariaLevel="4">
+
+- Switch must have an accessible name which is descriptive of its purpose.
+  - The visual name of the label must match the programmatic name.
+  <div aria-label="Example demo">
+      <SwitchLabel />
+  </div>
+
+  ```jsx
+  import Switch from 'terra-switch'; 
+
+    <Switch
+          isChecked={value}
+          labelText="Show Patients List in Menu"
+          onChange={setValue}
+        />
+  ```
```

Comment:
```suggestion
  ```jsx
  import Switch from 'terra-switch'; 

    <Switch
         isChecked={value}
         labelText="Show Patients List in Menu"
         onChange={setValue}
     />
  ```
```